test/extensions/INTEL/SPV_INTEL_task_sequence/sret-get.ll

```diff
@@ -0,0 +1,88 @@
+; RUN: llvm-as %s -o %t.bc
```

Comment:
It would be helpful to have a comment at the beginning of the test explaining the purpose.  Something like:

// Test translation of __spirv_TaskSequenceGetIntel with a sret (structure return) parameter.

Otherwise the reader will have to look at the commit history or try to derive the purpose from the filename and CHECK lines.